reference/typha/prometheus.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Prometheus statistics
+description: Review metrics for the Typha component if you are using Prometheus. 
+canonical_url: '/reference/typha/prometheus'
+---
+
+Typha can be configured to report a number of metrics through Prometheus.  See the
+[configuration reference](configuration) for how to enable metrics reporting.
+
+## Metric reference
+
+#### Typha specific
+
+Typha exports a number of Prometheus metrics.  The current set is as follows.  Since some metrics
+are tied to particular implementation choices inside Typha we can't make any hard guarantees that
+metrics will persist across releases.  However, we aim not to make any spurious changes to
+existing metrics.
+
+| Name          | Description     |
+| ------------- | --------------- |
+| `typha_breadcrumb_block` | Count of the number of times Typha got the next Breadcrumb after blocking. |
```

Comment:
I'm not sure what a breadcrumb is - any chance of an explanation?

Reply:
Huh! you have arrived to the point of my confusion!. I think some of the explanations that we are providing might not be the quickest way for people who are unfamiliar with Calico code to understand.

With that being said I Think each breadcrumb is a struct that contains a snapshot of datastore when it was created and how it is going to retrieve the next breadcrumb and drop the current. However, It would be better to ask someone like @fasaxc to confirm this.

https://github.com/projectcalico/typha/blob/8008b786367397b80c9a2d373d9cb1ba756a9d54/pkg/snapcache/cache.go#L399-L409